govcd/api_vcd_test.go

```diff
@@ -331,7 +332,7 @@ func (vcd *TestVCD) infoCleanup(format string, args ...interface{}) {
 // Gets the two components of a "parent" string, as passed to AddToCleanupList
 func splitParent(parent string, separator string) (first, second, third string) {
 	strList := strings.Split(parent, separator)
-	if len(strList) < 2 && len(strList) > 3 {
+	if len(strList) < 2 || len(strList) > 3 {
```

Comment:
Small matter, but would be nice to have a comment above the function with example values for parent and separator and expected output. When you read this function in isolation, it makes you wonder what is it used for.

Reply:
Done.

Reply:
Thanks for updates and a great test!!